package.json

```diff
@@ -49,15 +49,15 @@
   },
   "license": "MIT",
   "dependencies": {
-    "stylelint-config-standard": "^25.0.0",
-    "stylelint-config-standard-scss": "^3.0.0"
+    "stylelint-config-standard": "^29.0.0",
+    "stylelint-config-standard-scss": "^6.1.0"
   },
   "devDependencies": {
     "jest": "^27.4.3",
     "standard": "^16.0.4",
-    "stylelint": "^14.6.1"
+    "stylelint": "^14.16.1"
   },
   "peerDependencies": {
-    "stylelint": "^14.4.0"
+    "stylelint": "^14.14.0 || ^15.0.0"
```

Comment:
The good news?

Aside from deprecation warnings, it looks like this repo is already Stylelint v15 compatible. Although removing the deprecation warnings + adopting Prettier (from [Migrating to 15.0.0](https://stylelint.io/migration-guide/to-15/)) is out of scope for this PR

I've followed the approach of `stylelint-order` by supporting both v14 and v15:

* https://github.com/hudochenkov/stylelint-order/releases/tag/6.0.2

Reply:
I've dropped the `peerDependency` range so we can look at Stylelint v15 (without warnings) in another PR